### PR TITLE
ipq806x: add leds support

### PR DESCRIPTION
**Pull request by Ansuel**
This is a copy of https://github.com/openwrt/openwrt/pull/4036 with leds support.

### CODE REVIEW

**Comment by Ansuel** [comment]
I refreshed this with a new implementation proposed upstream for offload trigger support with leds.
Will totally change but it's better than before for sure.

File: target/linux/ipq806x/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -87,10 +87,25 @@ ipq806x_setup_macs()
...
+{
+	local board="$1"
+
+	case "$board" in
+		ubnt,unifi-ac-hd)
+		ip link set eth0 name ethtmp
+		ip link set eth1 name eth0
+		ip link set ethtmp name eth1
+		;;
+	esac
```
[the remark this answers is not in the document]
Comment (markmentovai) [reply target/linux/ipq806x/base-files/etc/board.d/02_network:145]: Ansuel, this happened in preinit before in order to make failsafe work on the Ethernet port likely to be connected, the port labeled MAIN, the only one to support PoE in. Will it have the same effect in /etc/board.d?

[the remark this answers is not in the document]
Comment (Ansuel) [reply target/linux/ipq806x/base-files/etc/board.d/02_network:145]: I need to check but would be very good to add a comment to it just so someone in refactoring process doesn't cut it LOL
@adschm can help with this?

File: target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8062-wg2600hp3.dts [hunk 2]
```diff
@@ -475,18 +476,130 @@
...
 				reg = <0>;
+
+				leds {
+					lan1_led@0 {
+						reg = <0>;
+						color = <LED_COLOR_ID_GREEN>;
+						function = LED_FUNCTION_LAN;
+						function-enumerator = <1>;
+						linux,default-trigger = "offload-phy-activity";
+						linux,supported-offload-triggers = "rx-blink", "tx-blink", "ollision-blink",
```
Comment (e9hack) [inline target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8062-wg2600hp3.dts:487]: typo "ollision-blink" -> "collision-blink"

**Comment by Ansuel** [comment]
Handled here https://github.com/openwrt/openwrt/pull/4036 with major reorg and changes due to upstream request.

**Pull request closed without merging**